src/cli.rs

```diff
@@ -168,6 +170,16 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
                 .required(false)
                 .help("If specified, transactions are send in batches of specified size"),
         )
+        .arg(
+            Arg::with_name("prioritization-fees")
+                .long("prioritization-fees")
+                .value_name("UINT")
+                .min_values(1)
```

Comment:
Something like:
```rust
                .min_values(1)
                 .validator(is_valid_percentage)
```

This validator is defined in solana here https://github.com/solana-labs/solana/blob/1c5ac415775c09b4bdf6118d02e4c4323726b6e8/clap-v3-utils/src/input_validators.rs#LL252C8-L252C27 for clap-v3